source/administration/command-line-tools.rst

```diff
@@ -738,6 +738,37 @@ mattermost config validate
       .. code-block:: none
 
         bin/mattermost config validate
+	
+mattermost db init
+------------------
+
+  Description
+    Initializes the database for a given DSN (data name source), executes migrations, and loads custom defaults when specified.
```

Comment:
Just confirming "DSN" = "data name source"? I would expect the acronym to follow the words.

Reply:
Good question, @wiersgallak! Common convention often puts an acronym's expansion ahead of the acronym itself, while the expansion of an initialism (such as DSN) follows. 
- Expansion is what the term means when written out in full. 
- Acronyms are abbreviations that use initials, but the initials are pronounced as a word rather than speaking the individual letters aloud. 
- Initialisms are when you abbreviate a word to only its initials.

All this being said, if we tend to position expansions first in MM docs, I can update this for consistency. I do recommend keeping both a short and long version to avoid ambiguity since DSN can mean different things to different audiences. CC @justinegeffen

Reply:
Circling back on this -- We're aligned on documenting acronyms and initialisms such that the expansion comes ahead of the acronym/initialism itself.

Reply:
That's a really helpful breakdown, @cwarnermm. I'm 4/5 on having it as "data source name (DSN)" purely for consistency. I'd like to add that breakdown to the style guide at some stage because it's very helpful.

Reply:
Agreed, @justinegeffen  - Consistency with existing docs is best in this case. I'm not so sure that the differences between acronyms and initialisms is widely understood enough ;)